### DISCUSSION TITLE
Update Zigbee2MQTT LXC to latest-dev

### DISCUSSION THREAD

**Discussion by stefansoeberg** [tteck/Proxmox#111] [Q&A] [answered]
Is it possible to update to latest-dev, and if so, how?

The home assistant update menu is super delicious. But another solution is also fine

**Comment by tteck** [comment, marked as answer]
Yep, 
```
cd /opt/zigbee2mqtt
```
```
nano update.sh
```
Change to "git checkout dev"
```
git checkout HEAD -- npm-shrinkwrap.json
git fetch
git checkout dev # Change 'dev' to 'master' to switch back to the release version
git pull
npm ci
```
https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html

**Comment by aheath70** [comment]
Is this still valid? Looking at `/opt/zigbee2mqtt/update.sh` the checkout is for package-lock.json.

Using GIT is not my strong suit, so I could be completely missing the mark here and need some guidance on how to install the dev branch.

**Reply by tteck** [reply]
https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html

**Reply by aheath70** [reply]
Thanks for the quick reply. I thought there may have been a way to initiate this from your scripts. Clearly not. I just followed that article and it worked, so all good. 

Great work on all the scripts @tteck - I have used quite a few of them and they are extremely helpful :)

**Reply by tteck** [reply]
You have the option in the initial creation of the Zigbee2MQTT LXC container.
